guides/ecto.md

```diff
@@ -2,23 +2,27 @@
 
 > **Requirement**: This guide expects that you have gone through the [introductory guides](installation.html) and got a Phoenix application [up and running](up_and_running.html).
 
-Most web applications today need some form of data validation and persistence. In the Elixir ecosystem, we have Ecto to enable this. Before we jump into building database-backed web features, we're going to focus on the finer details of Ecto to give a solid base to build our web features on top of. Let's get started!
+Most web applications today need some form of data validation and persistence. In the Elixir ecosystem, we have [`Ecto`](https://hexdocs.pm/ecto/) to enable this. Before we jump into building database-backed web features, we're going to focus on the finer details of Ecto to give a solid base to build our web features on top of. Let's get started!
 
 Phoenix uses Ecto to provide builtin support to the following databases:
 
-* PostgreSQL (via [`postgrex`](https://github.com/elixir-ecto/postgrex))
-* MySQL (via [`myxql`](https://github.com/elixir-ecto/myxql))
-* MSSQL (via [`tds`](https://github.com/livehelpnow/tds))
+* PostgreSQL (via [`postgrex`](https://hexdocs.pm/postgrex/))
+* MySQL (via [`myxql`](https://hexdocs.pm/myxql/))
+* MSSQL (via [`tds`](https://hexdocs.pm/tds/))
+* ETS (via https://github.com/evadne/etso [`etso`](https://hexdocs.pm/etso/))
+
 
 Newly generated Phoenix projects include Ecto with the PostgreSQL adapter by default. You can pass the `--database` option to change or `--no-ecto` flag to exclude this.
 
-Ecto also provides support for other databases and it has many learning resources available. Please check out [Ecto's README](https://github.com/elixir-ecto/ecto) for general information.
+Ecto also provides support for other databases and it has many learning resources available. Please check out [Ecto's README file](https://github.com/elixir-ecto/ecto/blob/master/README.md) for general information.
 
 This guide assumes that we have generated our new application with Ecto integration and that we will be using PostgreSQL. The introductory guides cover how to get your first application up and running. For instructions on switching to MySQL, please see the [Using MySQL](#using-mysql) section.
 
 ## Using the schema and migration generator
 
-Now that we have Ecto and Postgres installed and configured, the easiest way to use Ecto is to generate an Ecto *schema* through the `phx.gen.schema` task. Ecto schemas are a way for us to specify how Elixir data types map to and from external sources, such as database tables. Let's generate a `User` schema with `name`, `email`, `bio`, and `number_of_pets` fields.
+[NOTE]: # (It will also be good to show two commands.  one to see if Posgresql is installed, and the minimum required version and another one for Ecto)
```

Comment:
Improvement.

Reply:
Feel free to send a separate PR to discuss this, probably doesn't belong to something doing copy edits.